bootcamp/tutorials/integration/integrate_with_phidata.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<h2>Use Phidata agent to answer a question</h2><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>